README.md

```diff
@@ -74,6 +74,29 @@ in Python, Julia, R, Rust, and C.
 
 Examples of other functionality can be found in the `test` folder for each interface.
 
+## Software Using BridgeStan
+
+The following are some projects we are aware of that use BridgeStan. If you know of others, please let us know!
```

Comment:
I'd just say:

> The following projects use BridgeStan. 

or if you absolutely must,

> We are aware of the following projects using BridgeStan.

though I don't think you need to given that you ask for others.